src/js/hooks/cycleCount/useProductsTab.jsx

```diff
@@ -85,7 +85,10 @@ const useProductsTab = ({
     errorMessageId: 'react.cycleCount.table.errorMessage.label',
     defaultErrorMessage: 'Unable to fetch products',
     // We should start fetching only after clicking the "Load Table" button
-    shouldFetch,
+    // or after refreshing the page with filters selected
```

Comment:
I'd add "with required filters selected", as I can't see you considering the product select here